<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Documentation</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="link_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background:rgb(85, 85, 255);
color:rgb(255, 255, 255);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Open Online Documentation at https://www.bit.ly/xxxxxx</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background:rgb(236, 236, 236);
padding:10px;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;With the raise of machine era. Data is costlier then knowledge. According to Mitchel definition of machine learning &amp;quot;A program is said to learn from experience E with respect to some task T and some performance measure P if its performance on T as measured by P improves with experience E&amp;quot;. Letâ€™s think about the experience. What if, the experience is not correct (i.e. noisy sample) or the quality of input experience is not so good? Definitely it will effect on P. to improve the P with every E we must have to provide correct E. for that we are going to create a new dataset for machine learning which contains label human faces. That dataset will contain at least 10 face sample each form front, left and right. The sample may or may not contains any facial expression or other situation. We are going to click all image in normal sunlight (or natural condition). &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>textBrowser</zorder>
  <zorder>link_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
